### PR TITLE
docs: update optimization and source map section

### PR DESCRIPTION
**Pull request by alan-agius4**
With this change we split the optimization and source map configuration section section into two, improve it overall by adding the options a table and add the new options (`inlineCritical` and `minify`) that have been introduced in CLI version 11.1

### CODE REVIEW

**Comment by petebacondarwin** [comment]
Here is a link to the preview of the changed guide: https://pr40037-809fbd2.ngbuilds.io/guide/workspace-config

**Comment by alan-agius4** [comment]
Link to guide  https://pr40037-5bde243.ngbuilds.io/guide/workspace-config

**Review by aikithoughts — APPROVED** [review]
Reviewed-for: global-docs-approvers

File: aio/content/guide/workspace-config.md [hunk 1]
```diff
@@ -361,34 +361,156 @@ Files in that folder, such as `src/style-paths/_variables.scss`, can be imported
...
+<td><code>hidden</code></td>
+<td>Output source maps used for error reporting tools.</td>
+<td><code class="no-auto-link">boolean</code></td>
+<td><code>false</code></td>
+</tr>
+</tbody>
+</table>
+
+
+You can supply a value such as the following to output source maps to one or the other:
```
Comment (kyliau) [inline aio/content/guide/workspace-config.md:513]:
```suggestion
The example below shows how to toggle one or more values to configure the source map outputs:
```

**Review by kyliau — CHANGES_REQUESTED** [review]

File: aio/content/guide/workspace-config.md [hunk 2]
```diff
@@ -361,34 +361,156 @@ Files in that folder, such as `src/style-paths/_variables.scss`, can be imported
...
-
-* The `optimization` option applies to scripts, styles and fonts. You can supply a value such as the following to apply optimization to one or the other:
+### Optimization configuration
+
+The `optimization` browser builder option can be either a Boolean or an Object for more fine-grained configuration. This option enables various optimizations of the build output.
+Including minification of scripts and styles, tree-shaking, dead-code elimination, inlining of critical CSS and fonts inlining.
+
+There are several options that can be used to fine-grained the optimization of an application.
+
+<table class="is-full-width is-fixed-layout">
```
[the remark this answers is not in the document]
Comment (alan-agius4) [reply aio/content/guide/workspace-config.md:371]: I don't know, I just followed the style of other tables in AIO.

File: aio/content/guide/workspace-config.md [hunk 3]
```diff
@@ -361,34 +361,156 @@ Files in that folder, such as `src/style-paths/_variables.scss`, can be imported
...
-
-### Optimization and source map configuration
-
-The `optimization` and `sourceMap` browser builder options can be either a Boolean or an Object for more fine-grained configuration.
-In this section we will explain how to fine tune these options.
-
-* The `optimization` option applies to scripts, styles and fonts. You can supply a value such as the following to apply optimization to one or the other:
+### Optimization configuration
+
+The `optimization` browser builder option can be either a Boolean or an Object for more fine-grained configuration. This option enables various optimizations of the build output.
```
[the remark this answers is not in the document]
Comment (alan-agius4) [reply aio/content/guide/workspace-config.md:366]: Much better!

**Comment by alan-agius4** [comment]
Link to guide https://pr40037-e1c5fb9.ngbuilds.io/guide/workspace-config

**Review by kyliau — APPROVED** [review]

**Pull request closed without merging**